lms/static/lms/js/require-config.js

```diff
@@ -49,7 +49,12 @@
             "backbone.paginator": "js/vendor/backbone.paginator.min",
             "underscore": "js/vendor/underscore-min",
             "underscore.string": "js/vendor/underscore.string.min",
-            "jquery": "js/vendor/jquery.min",
+            // The jquery-migrate library was added in upgrading from
+            // jQuery 1.7.x to 2.2.x.  This config allows developers
+            // to depend on "jquery" which opaquely requires both
+            // libraries.
+            "_jquery": "js/vendor/jquery.min",
```

Comment:
We need to make sure the right thing is happening with optimized LMS files on a sandbox. I would expect us to need to define _jquery above in the "defineDependency" section to avoid jquery-migrate from showing up as an uncached file. jquery-migrate will already be present on the page before RequireJS loads in production because it is in base_vendor.